src/main/java/frc/robot/commands/DriveCommand.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import java.util.function.BooleanSupplier;
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.DriveTrain;
+
+public class DriveCommand extends CommandBase {
+  /** Creates a new DriveCommand. */
+
+  DriveTrain driveTrain;
```

Comment:
You missed our discussion on Zoom. We want to enforce the convention of using "m_" as a prefix to class member variables. For example, this should be "DriveTrain m_driveTrain;". Then Line 21 would be "m_driveTrain = driveTrain;"